PortalFetch/crawler_download.py

```diff
@@ -257,50 +244,47 @@ def main():
     Login in De Anza myportal using username and password.
     click Apps-Lookup Classes-Select by term -submit-Advanced Search-in Subject, select all-Section search-Download all the course infromation-Save in an excel
     """
-    driver = webdriver.Chrome(ChromeDriverManager().install())
-    login_myportal(driver)
-
-    # Wait for the 'list-group-item' can be found and clicked
-    web_driver_counter = 400
-    list_group_item = None
-    while web_driver_counter:
+    quartervalue = parser.get('config', 'quarter_value')
+    quartervalueList = quartervalue.split('_')
```

Comment:
It is better to use ',' to represent a list of values, it should be a quick fix since all other functionalities are working, please remember to trim after `split(',')` since users could add spaces after comma